pageserver/src/metrics.rs

```diff
@@ -96,6 +96,16 @@ static CURRENT_PHYSICAL_SIZE: Lazy<UIntGaugeVec> = Lazy::new(|| {
     .expect("failed to define a metric")
 });
 
+#[cfg(not(test))]
+pub(crate) static REMOTE_PHYSICAL_SIZE: Lazy<UIntGaugeVec> = Lazy::new(|| {
```

Comment:
Right.

So, for deletes, should we change the metric as soon as we've uploaded the new index part? Otherwise the metric is ahead, and imagine those deletes get stuck for some reason.

Reply:
Hm, we could do that for layer file deletion.

But for layer file upload, the file upload is separate from the index part update.
So, in `schedule_index_upload`, we don't get the delta.

Then again, we get the full IndexPart.

So, we could just .set() the metric there.

I'll try that.

Reply:
@LizardWizzard see my last push from yesterday.